static/js/impala/suggestions.js

```diff
@@ -130,12 +130,14 @@ $.fn.searchSuggestions = function($results, processCallback, searchType) {
             $results.filter('.visible').removeClass('visible');
             return;
         }
+        var urlVal = encodeURIComponent($self.val());
```

Comment:
Looking at this a bit more, since it's using attr for the href the HTML escaping is not required not sure why I was thinking this was using `html()` but that's only used here: https://github.com/mozilla/addons-server/blob/master/static/js/impala/site_suggestions.js#L13